### ISSUE TITLE
Invitation en mailer deamon

### ISSUE THREAD

**Issue by webaaz** [afup/aperophp#152]
J'ai reçu une invitation google calendar pour l'apero php.
Quand je dis, ok je viens, il me renvoie un mailer deamon :-/

**Comment by mikaelrandy** [comment]
@agallou il semblerais que ça déconne avec Google Calendar parce que l'adresse de reply est noreply@aperophp.net

**Comment by agallou** [comment]
@mikaelkael il y a une adresse qu'on pourrait utiliser qui ne renverrais pas d'erreur ?

**Comment by webaaz** [comment]
Une adresse qu'il faudrait écouter pour changer le statut : sur de venir !
Peut être ou ne viens pas pour gérer les mises à jour :)
Le 15 déc. 2015 11:30 AM, "Adrien Gallou" <notifications@github.com> a
écrit :

**Comment by mikaelkael** [comment]
Dites moi : apero@aperohphp.net, robot@aperophp.net, jai-soif@aperophp.net...

**Comment by mikaelrandy** [comment]
J'aime bien la dernière proposition

Le mar. 15 déc. 2015 à 14:58, Mickael Perraud <notifications@github.com> a
écrit :

> Dites moi : apero@aperohphp.net, robot@aperophp.net, jai-soif@aperophp.net
> ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/afup/aperophp/issues/152#issuecomment-164772896>.
>
-- 
Mikael Randy

**Comment by mikaelrandy** [comment]
En fait, en regardant l'email reçu, le soucis semble être que l'adresse à laquelle Google Calendar pense avoir envoyé l'invitation est l'adresse no-reply@aperophp.net.

Le soucis vient certainement de là.

**Comment by mikaelrandy** [comment]
@mikaelkael est-ce que tu avais créé cette adresse ?

Tu pourrais me donner les identifiants par email ?

**Comment by mikaelkael** [comment]
Oui un adresse "noreply@aperophp.net" a bien été créée le 14 décembre. Mais pas "no-reply@"

**Comment by mikaelrandy** [comment]
@mikaelkael je ne retrouve pas les identifiants.
Tu peux me les transmettre par email ?

**Comment by mikaelkael** [comment]
@mrandy : quels identifiants ?

**Comment by mikaelrandy** [comment]
@mikaelkael les identifiants de connexions à la boite mail.
Que je puisse faire un cron qui lise les emails pour marquer la participation à l'apero

**Comment by mikaelkael** [comment]
Ah ! Il n'y pas de compte pop. Juste un alias pour que l'adresse soit acceptée. Tu veux un compte pop en fait ?

**Comment by mikaelrandy** [comment]
Oui. Si on envoie une invitation et qu'on permet d'y répondre, il faut qu'on gère cette réponse.
Du coup, il faudrait qu'une boite reçoive ces réponses, pour pouvoir traiter les emails et modifier les participations.
